server/src/test/java/org/opensearch/action/admin/cluster/node/stats/NodeStatsTests.java

```diff
@@ -36,8 +36,8 @@
 import org.opensearch.common.io.stream.BytesStreamOutput;
 import org.opensearch.common.io.stream.StreamInput;
 import org.opensearch.discovery.DiscoveryStats;
-import org.opensearch.discovery.zen.PendingClusterStateStats;
-import org.opensearch.discovery.zen.PublishClusterStateStats;
+import org.opensearch.cluster.coordination.PendingClusterStateStats;
```

Comment:
Interesting, it seems like `PendingClusterStateStats` is never actually used (in a meaningful way). The only place it is instantiated is https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/cluster/coordination/Coordinator.java#L863, but besides that - this stats is never updated. Am I missing something or it could be dropped as well?

Reply:
Yeah, looks pretty weak to me as well. Let's address that in a follow up issue/PR?

Reply:
👍